src/styles/overrides/_fancybox.scss

```diff
@@ -0,0 +1,25 @@
+.fancybox-title-float-wrap {
+  position: absolute;
+  bottom: 0;
+  right: 50%;
+  margin-bottom: -35px;
+  z-index: 8050;
```

Comment:
@tmccoy529 I'm testing the styles as they are on prod (if they exist in some way in beta let me know) and not really seeing the z-index add any meaningful functionality. When removed, the text doesn't seem to disappear behind anything. Are you getting similar results? I'd test again on beta, but if the results are the same, this line can probably be removed.